### ISSUE TITLE
Error when single GPU, no pre-trained model is used

### ISSUE THREAD

**Issue by 17648145240** [xzz777/SCTNet#8]
Hi author, your SCTNet has inspired me a lot, and I would really like to reproduce your method, I need to train the network on my own dataset (dataset with the same directory structure as CitySpaces and previously trained successfully with other networks).
In order to run successfully on my single GPU, I made the following changes to your multi-GPU code:
(1) I commented out the attributes of backbone and auxiliary_head in the config file:
 init_cfg=dict(
        type='Pretrained', # checkpoint=checkpoint_cfg=dict(
        checkpoint=checkpoint_backbone),
and changed it to: init_cfg=dict(type=None),
(2) Modify all nn.SyncBatchNorm( ) on backbone and Head to nn.BatchNorm2d( ).
But still got the error：

![Uploading image.png…]()

Do you know the cause of this error please? Or how can I modify your code for tasks with single GPU and without pre-trained models?
If anyone has such code running successfully, can you share a copy with me? Thanks a lot!
My e-mail is 17648145240@163.com
